model/countries/AU/AU-model.yaml

```diff
@@ -0,0 +1,23 @@
+cut-off-children:  # don't expose nodes below these levels
+  - family-name
+
+cut-off-tokens:
+  - honorific-prefix
+  - honorific-suffix
+  - admin-area2
+  - admin-area3
+  - admin-area4
+  - locality2
+  - locality3
+  - locality4
+  - cross-streets
+  - building-name
```

Comment:
This is still listed in AU.html. Did you regenerate it?

I would recommend adding `- building` to `cut-off-children:`

Reply:
Added both `-building` and `-street` to cut-offs and regenerated AU.html. PTAL!